New mac installer (#296)
jcsteh/osara: readme.md

Please follow these instructions to install it:

1. Open the OSARA disk image file you downloaded.

> 2. Open the "Install OSARA.command" file.  On macOS Catalina and later, you have to choose open from the context menu, accessed with VO+Shift+M.

Reviewer: ```suggestion
2. Open the "Install OSARA.command" file.
 On macOS Catalina and later, you have to choose open from the context menu, accessed with VO+Shift+M.
```

> 3. Follow the instructions.  If you wish to replace the existing keymap with the OSARA keymap, (which is recommended), answer yes when prompted. A backup of your existing keymap will be made in Reaper's KeyMaps folder. 

Reviewer: ```suggestion
3. Follow the instructions.
 If you wish to replace the existing key map with the OSARA key map (which is recommended), answer yes when prompted.
  A backup of your existing key map will be made in Reaper's KeyMaps folder. 
```

> 4. Press command+e to eject the disk image.
> 5. The installer leaves a terminal window open. it can be closed with Command+Q.

Reviewer: ```suggestion
4. The installer leaves a terminal window open.
 It can be closed with Command+Q.
5. Press command+e to eject the disk image.
```
